jenkins/docker/javascript.Dockerfile

```diff
@@ -27,6 +27,11 @@ RUN apt-get install -y python3-pip
 RUN apt-get install -y shellcheck \
 	&& pip install gitlint
 
+# codecov uploader
+RUN curl -Os https://uploader.codecov.io/latest/linux/codecov \
+	&& chmod +x codecov \
+	&& mv codecov /usr/local/bin
```

Comment:
is this not in apt-get repo ?

Reply:
No, they had an older version I think in apt.
https://docs.codecov.com/docs/codecov-uploader